bundles/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/core/TelldusCoreBridgeHandler.java

```diff
@@ -168,8 +177,9 @@ public void rescanTelldusDevices() {
     }
 
     private synchronized void setupListeners() {
-        eventHandler = new TellstickEventHandler(deviceList);
+        TellstickEventHandler eventHandler = new TellstickEventHandler(deviceList);
```

Comment:
Optional, could be shortened:
```suggestion
        TellstickEventHandler eventHandler = this.eventHandler = new TellstickEventHandler(deviceList);
```
(by removing line 182)